src/EditorFeatures/CSharpTest/MakeLocalFunctionStatic/MakeLocalFunctionStaticTests.cs

```diff
@@ -50,6 +52,27 @@ static int fibonacci(int n)
 parseOptions: CSharp8ParseOptions);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestWithOptionOff()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        int [||]fibonacci(int n)
+        {
+            return n <= 1 ? n : fibonacci(n - 1) + fibonacci(n - 2);
+        }
+    }
+}",
+new TestParameters(
+    parseOptions: CSharp8ParseOptions,
+    options: Option(CSharpCodeStyleOptions.PreferStaticLocalFunction, CodeStyleOptions.FalseWithSilentEnforcement)));
```

Comment:
Not sure this test belongs with this PR. Probably want to revert changes to this file.

Reply:
I'd prefer to keep it.  I don't want ot make another PR just to improve test coverage for this feature :)